src/styles/cards.scss

```
@@ -0,0 +1,172 @@
+.page {
+
+  &__catalog {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 72px;
+  }
+}
+
+.catalog {
+
```

remove unnecessary lines
```suggestion
  &__catalog {
    display: flex;
    justify-content: center;
    margin-bottom: 72px;
  }
}

.catalog {
```